e2e-tests/cypress/support/core-page-checks.js

```diff
@@ -87,9 +91,15 @@ const corePageChecks = ({
     checkBackLink(currentHref, backLink);
   }
 
+  if (assertCookies) {
   // check analytics cookie banner
-  cy.checkAnalyticsCookiesConsentAndAccept();
-  cy.rejectAnalyticsCookies();
+    cy.checkAnalyticsCookiesConsentAndAccept();
+    cy.rejectAnalyticsCookies();
+  }
+
+  if (assertServiceHeading) {
+    cy.checkHeaderServiceNameAndHref(isInsurancePage);
+  }
```

Comment:
is there an instance where we would *not* be asserting the service heading?

Reply:
page not found.  A seperate test for the heading there as it uses generic and not quote